config/routes.rb

```diff
@@ -3,4 +3,5 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root "blog#index"
```

Comment:
This looks like it is setting root to index, but I don't see a route for your index.